server/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -900,6 +907,21 @@ protected final RefreshResponse refresh(String... indices) {
         return actionGet;
     }
 
+    /**
+     * Returns <code>true</code> iff the given index exists otherwise <code>false</code>
+     */
+    protected static boolean indexExists(String index) {
+        return indexExists(index, client());
+    }
+
+    /**
+     * Returns <code>true</code> iff the given index exists otherwise <code>false</code>
+     */
+    public static boolean indexExists(String index, Client client) {
+        ClusterStateResponse state = client.admin().cluster().prepareState().get();
```

Comment:
The original version uses the GetIndex api which has been removed, this is a workaround.